rust/src/nasl/syntax/metadata.rs

```diff
@@ -0,0 +1,53 @@
+use crate::nasl::syntax::parser::ErrorKind;
+use crate::nasl::syntax::parser::Result;
+
+use super::grammar::Ast;
+use super::grammar::Atom;
+use super::grammar::Expr;
+use super::grammar::Statement;
+use super::parser::Error;
+use super::{grammar::If, parser::Parse};
+
+pub struct DescriptionBlock {
+    stmts: Vec<Statement>,
+}
+
+impl DescriptionBlock {
+    pub(crate) fn to_ast(self) -> Ast {
+        Ast::new(self.stmts)
+    }
+}
+
+impl Parse for DescriptionBlock {
+    fn parse(parser: &mut super::Parser) -> Result<Self> {
+        let mut if_: If = parser.parse()?;
+        if if_.else_branch.is_some() {
+            return Err(ErrorKind::InvalidDescriptionBlock(
```

Comment:
question: why is this considered an error? From a script perspective it just means that it is not an description run, so if an script author wants to have a else branch and work in there why should we not allow that?